templates/flavors/aks/cluster-template.yaml

```diff
@@ -74,6 +74,7 @@ spec:
   mode: System
   osDiskSizeGB: 512
   sku: "${AZURE_NODE_MACHINE_TYPE}"
+  osDiskType: "Managed"
```

Comment:
These are temporary changes so that we can run tests before merging. Once we have passing signal I'll remove them.